ports/dbus/portfile.cmake

```
@@ -0,0 +1,64 @@
+vcpkg_from_gitlab(
+    GITLAB_URL https://gitlab.freedesktop.org/
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO dbus/dbus
+    REF c91ca6edad658274607323a438eea7c7c6c5e392 #1.13.18
+    SHA512  4dd4d369152591040ebe9f474a0ba8911d8a91546d64b1d6f7335b7fd8026bd99a8a4fe1c78b80eb2e31e9e58324d432857e2a7af1d1cb950d22b4430cc0f7ac
+    HEAD_REF master # branch name
+    PATCHES 
+        cmake.dep.patch #patch name
+        rt_pc_link.patch
+        pkgconfig.patch
+        getpeereid.patch # missing check from configure.ac
+        rdynamic.patch # OSX doesn't like '-Wl,--export-dynamic'
+) 
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DDBUS_BUILD_TESTS=OFF
+        -DDBUS_ENABLE_XML_DOCS=OFF
+        -DDBUS_INSTALL_SYSTEM_LIBS=OFF
+        -DDBUS_ENABLE_PKGCONFIG=ON
+        #-DDBUS_SERVICE=ON
+        -DDBUS_WITH_GLIB=ON
+        -DXSLTPROC_EXECUTABLE=FALSE
+        -DENABLE_SYSTEMD=ON
+        -DDBUS_ENABLE_PKGCONFIG=ON
+        -DPKG_CONFIG_FOUND=TRUE
+        "-DCMAKE_INSTALL_SYSCONFDIR=${CURRENT_PACKAGES_DIR}/etc/${PORT}"
+        "-DWITH_SYSTEMD_SYSTEMUNITDIR=lib/systemd/system"
+        "-DWITH_SYSTEMD_USERUNITDIR=lib/systemd/user"
+)
+vcpkg_cmake_install()
+vcpkg_copy_pdbs()
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/DBus1")

```

```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME "DBus1" CONFIG_PATH "lib/cmake/DBus1")
```

we should probably teach vcpkg_cmake_config_fixup to use the correct subfolder automatically (or at least check it)